requirements.txt

```diff
@@ -1,4 +1,5 @@
-mona-sdk
-alt-profanity-check
-phonenumbers
-tiktoken
\ No newline at end of file
+mona-sdk>=0.0.49
```

Comment:
People are already started to use the SDK. Don't we need to increase the version number of our client?